.github/workflows/smoke-test-pr-check.yaml

```diff
@@ -71,11 +63,25 @@ jobs:
           sed -i -r -e "s|${REPLACE_FROM}||" build/dockerfiles/assembly.copy.Dockerfile
           REPLACE_FROM="COPY --from=linux-musl-content --chown=0:0 /checode-linux-musl /mnt/rootfs/checode-linux-musl"
           sed -i -r -e "s|${REPLACE_FROM}||" build/dockerfiles/assembly.copy.Dockerfile
+          
           export DOCKER_BUILDKIT=1
-          docker build -f build/dockerfiles/assembly.copy.Dockerfile -t che-code .
+          docker build --platform linux/amd64 -f build/dockerfiles/assembly.copy.Dockerfile -t che-code .
```

Comment:
Just a minor question, for my better understanding.
Isn't `--platform linux/amd64` redundant here and above?
Considering the `smoke-test` job is running on `ubuntu-22.04`.

Reply:
Cleaned it up.